yt_dlp/extractor/tenplay.py

```diff
@@ -35,26 +37,28 @@ class TenPlayIE(InfoExtractor):
     _GEO_BYPASS = False
 
     def _get_bearer_token(self, video_id):
-        _authdata = self._get_login_info()
-        _time = datetime.now()
-        _auth_header = base64.b64encode(f'{_time.year}{"{:02d}".format(_time.month)}{"{:02d}".format(_time.day)}000000'.encode('ascii')).decode('ascii')
-        if(_authdata[0] is None or _authdata[1] is None):
-            raise Exception('Your 10play account\'s details must be provided with --username and --password.')
         username, password = self._get_login_info()
+        _time = datetime.now()
+        _timestamp = str(_time.year) + str("{:02d}".format(_time.month)) + str("{:02d}".format(_time.day)) + '000000'
+        _auth_header = base64.b64encode(_timestamp.encode('ascii')).decode('ascii')
+        if username is None or password is None:
+            self.raise_login_required('Your 10play account\'s details must be provided with --username and --password.')
```

Comment:
can be simplified to:
```suggestion
        if username is None or password is None:
            self.raise_login_required('Your 10play account\'s details must be provided with --username and --password.')
        _timestamp = datetime.now().strftime('%Y%M%d000000')
        _auth_header = base64.b64encode(_timestamp.encode('ascii')).decode('ascii')
```